lib/licenses.nix

```diff
@@ -1,44 +1,56 @@
 { lib }:
-let
 
-  spdx = lic: lic // {
-    url = "https://spdx.org/licenses/${lic.spdxId}.html";
-  };
-
-in
-
-lib.mapAttrs (n: v: v // { shortName = n; }) ({
+lib.mapAttrs (lname: lset: let
+  defaultLicense = rec {
+    shortName = lname;
+    free = true; # Most of our licenses are Free, explicitly declare unfree additions as such!
```

Comment:
It's mostly the spdx licenses that are free, so would `free = lset ? spdxId` (free if license has spdxId set) make sense?

Reply:
I guess it would but that would also mean any Free license that doesn't have an spdx identifier would be unfree by default which I'm not sure we'd want.

Reply:
I'm going to assert that this is fine. If you have a concern please open up an issue or send a PR and we can discuss the alternative.